init.g

```diff
@@ -46,7 +46,9 @@ ReadPackage("GAPDoc", "lib/Examples.gd");
 
 
 # try to find terminal encoding
-GAPInfo.tmpfunc := function()
+CallFuncList(function()
```

Comment:
You are right, `GAPInfo` isn't read-only. In fact I thought I had removed the bogus commit message claiming this incorrectly, but I see now that my push to do so failed as I forgot to use `--force`... will correct that ASAP.